keen/utilities.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+VERSION = None
+
+def version():
+  """
+  Retrives the current version of the SDK
+  """
+
+  global VERSION
+  if VERSION is None:
+    version_file = open(os.path.join('.', 'VERSION'))
+    VERSION = version_file.read().strip()
+
+  return VERSION
+
+def headers(api_key):
+  """
+  Helper function to easily get the correct headers for an endpoint
+
+  :params api_key: The appropriate API key for the request being made
+  """
+
+  return {
+      "Content-Type": "application/json",
+      "Authorization": api_key,
+      "Keen-Sdk": "python-{0}".format(version())
```

Comment:
let's confirm capitalization. `Keen-SDK` vs `Keen-Sdk`. (i prefer the former but don't really care overall. let's just make sure we match! :) )